src/components/Rover.tsx

```diff
@@ -1,15 +1,36 @@
-import { RoverManifestDetails } from "../api/nasaApi";
+import { useEffect, useState } from "react";
+import { RoverManifestDetails, getRoverManifestData } from "../api/nasaApi";
+import "./Rover.scss";
 
 interface RoverProps {
-  roverDetails: RoverManifestDetails;
+  rover: string;
 }
 
-const Rovers = ({ roverDetails: roversDetails }: RoverProps) => {
+const Rover = ({ rover }: RoverProps) => {
+  const [roverData, setRoverData] = useState<RoverManifestDetails>();
+
+  useEffect(() => {
+    setRoverData(undefined);
+    getRoverManifestData(rover).then((data) => setRoverData(data));
+  }, [rover]);
+
   return (
-    <div className="Rovers">
-      {roversDetails ? <div></div> : <p>Loading...</p>}
+    <div className="Rover">
+      {roverData !== undefined ? (
+        <>
+          <div>
```

Comment:
You need _either_ the `<>` _or_ the `<div>`, but not both! I recommend only the `<>`

Reply:
we took the <> ones instead because removing the <div> was changing the layout of our camera details.